### PR TITLE
DDUG frederick

### LINKED ISSUE #123: Update UG with vis and watchlist

### PR DESCRIPTION
**Pull request by wwweert123**
fixes #123

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -4,27 +4,27 @@
 
 **Xchart (A Simple Charting Library for Java)**
 - author: KNOWN
-- source: https://knowm.org/open-source/xchart/
+- source: [https://knowm.org/open-source/xchart/]()
 
 **JSON Simple (simple Java toolkit for encoding and decoding JSON)**
 - author: Yidong Fang (Google Code)
-- source: https://code.google.com/archive/p/json-simple/
+- source: [https://code.google.com/archive/p/json-simple/]()
```
Comment (ryan1604) [inline docs/DeveloperGuide.md:11]: Thanks for adding link:D

File: docs/UserGuide.md [hunk 2]
```diff
@@ -154,6 +158,73 @@ Example output:
...
+
+Example of usage: `vis /t income /c pie`
+
+Example of output:
+
+```
+Displaying piechart for expense
+```
+
+![](images/vis/visOutput.png)
```
Comment (ryan1604) [inline docs/UserGuide.md:226]: Very nice

**Review by ryan1604 — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (NeoMinWei) [reply docs/DeveloperGuide.md:11]: walao never add for me

File: docs/UserGuide.md [hunk 3]
```diff
@@ -154,6 +158,73 @@ Example output:
...
+
+```
+You have successfully deleted: 
+Meta Platforms Inc - Class A
+Use watchlist command to view updated Watchlist
+```
+
+### Visualizing your cashflow: `vis`
+
+Using this command to visualize your income or expenses in a pie chart or bar char
```
Comment (NeoMinWei) [inline docs/UserGuide.md]: bar char

File: docs/UserGuide.md [hunk 4]
```diff
@@ -154,6 +158,73 @@ Example output:
 Balance: 3790.00
 ```
 
+### Viewing Watchlist: `watchlist`
+
+View your current watchlist with stocks that you are interested in
```
Comment (NeoMinWei) [inline docs/UserGuide.md]: maybe can put on what exchange the watchlist is displaying

File: docs/UserGuide.md [hunk 5]
```diff
@@ -13,6 +13,10 @@
     * [Viewing budget](#viewing-budget-budget-view)
   * [Displaying Overview](#displaying-overview-overview)
   * [View Balance](#viewing-balance-balance)
+  * [WatchList](#viewing-watchlist-watchlist)
+    * [Adding Stock](#adding-stock-to-watchlist-addstock)
+    * [Deleting Stock](#deleting-budget-budget-delete)
+  * [Visualization](#Visualization)
   * [Exiting the program](#exiting-the-program-exit)
   * [Saving data](#saving-the-data)
   * [Loading data](#loading-the-data)
```
Comment (NeoMinWei) [inline docs/UserGuide.md:22]: very cool in adding the links

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 3 above)
Comment (wwweert123) [reply docs/UserGuide.md]: thanks for seeing my mistakes!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (wwweert123) [reply docs/DeveloperGuide.md:11]: NP

File: docs/UserGuide.md (hunk 4 above)
Comment (wwweert123) [reply docs/UserGuide.md]: yes it should be included in the output already

File: docs/UserGuide.md (hunk 5 above)
Comment (wwweert123) [reply docs/UserGuide.md:22]: Thank you!

**Pull request merged**